### PR TITLE
Fix bug of plain object

### PR DESCRIPTION
**Pull request by VitorDaynno**
- [Erro na geração de JWT](https://trello.com/b/IjmAUumz/rpg)

### CODE REVIEW

**Comment by VitorDaynno** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Full-Moon-Castle/werewolf/pullRequest?prid=5345778&bid=17501983)

File: src/helpers/jwtHelper.js [hunk 1]
```diff
@@ -55,11 +55,17 @@ class JWTHelper {
...
-      const token = jwt.sign( entity, jwtSecret, { expiresIn });
+      const token = jwt.sign( {
+        id,
+        email,
+        createdDate: created_date,
+      }, jwtSecret, { expiresIn });
 
       return token;
     } catch (errors) {
+      console.log(errors)
```
Comment (VitorDaynno) [inline src/helpers/jwtHelper.js:68]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/gh/Full-Moon-Castle/werewolf/pullRequest?prid=5345778)

File: src/helpers/jwtHelper.js [hunk 2]
```diff
@@ -55,11 +55,17 @@ class JWTHelper {
 
   createToken(entity) {
     try {
+      const {  id, email, created_date } = entity;
```
Comment (VitorDaynno) [inline src/helpers/jwtHelper.js:58]: Codacy Issue found: [Identifier 'created_date' is not in camel case.](https://app.codacy.com/gh/Full-Moon-Castle/werewolf/pullRequest?prid=5345778)

Comment (VitorDaynno) [inline src/helpers/jwtHelper.js:58]: Codacy Issue found: [Multiple spaces found before 'id'.](https://app.codacy.com/gh/Full-Moon-Castle/werewolf/pullRequest?prid=5345778)

File: src/helpers/jwtHelper.js (hunk 1 above)
Comment (VitorDaynno) [inline src/helpers/jwtHelper.js:68]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/manual/VitorDaynno/werewolf/pullRequest?prid=5345779)

**Comment by VitorDaynno** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VitorDaynno/werewolf/pullRequest?prid=5345779&bid=17501984)

File: src/helpers/jwtHelper.js (hunk 2 above)
Comment (VitorDaynno) [inline src/helpers/jwtHelper.js:58]: Codacy Issue found: [Multiple spaces found before 'id'.](https://app.codacy.com/manual/VitorDaynno/werewolf/pullRequest?prid=5345779)

Comment (VitorDaynno) [inline src/helpers/jwtHelper.js:58]: Codacy Issue found: [Identifier 'created_date' is not in camel case.](https://app.codacy.com/manual/VitorDaynno/werewolf/pullRequest?prid=5345779)

File: src/helpers/jwtHelper.js (hunk 1 above)
Comment (cesarmacedo) [inline src/helpers/jwtHelper.js]: Remover console.log por favor

**Review by cesarmacedo — APPROVED** [review]

**Pull request merged**